lms/djangoapps/learner_dashboard/tests/test_views.py

```diff
@@ -114,18 +120,19 @@ def test_api_returns_discussions_iframe(self):
         self.assertIsInstance(response.data['discussion']['iframe'], Markup)
         self.assertIn('iframe', str(response.data['discussion']['iframe']), )
 
-    def test_program_does_not_exist(self):
+    def test_program_empty_response(self):
```

Comment:
`test_program_without_provider_config`

Reply:
This is not what is being tested. Testing empty response in case the user does not have permission here

Reply:
permission in case of enrollment? or any other access please include that into the docstring.